flow/project.py

```diff
@@ -756,6 +760,12 @@ def detect_operation_graph(self):
         output graph could be visualized using Matplotlib and NetworkX with the
         following code:
 
+        Raises a ``RuntimeError`` if a condition does not have a tag. This can
```

Comment:
This block interrupts a code sample. "with the following code: ... [code]" Move it below that code sample. (Or maybe above the previous paragraph, but I think I prefer below.)

Reply:
Good catch, should have paid more attention.